src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -367,6 +367,12 @@ def conn_wrapper(self, fs_handle, **kwargs):
             return result
         return conn_wrapper
 
+    def nametojson(self, names):
+        dict = []
```

Comment:
You're naming a list as a `dict`? This is confusing. Please give a better name, and preferably not just `list`.